lib/membrane_live/chats.ex

```diff
@@ -27,19 +53,60 @@ defmodule MembraneLive.Chats do
     |> Repo.all()
   end
 
-  def add_chat_message(event_id, user_name, user_email, is_auth, content, time_offset) do
-    if is_auth do
-      add_authenticated_chat_message(event_id, user_email, content, time_offset)
+  @spec add_chat_message(event_id(), user(), String.t()) :: channel_message()
+  def add_chat_message(event_id, user, content) do
+    key =
+      if user.is_auth and user.is_presenter, do: :start_timestamps, else: :client_start_timestamps
+
+    offset =
+      case :ets.lookup(key, event_id) do
```

Comment:
Isn't it a typo here? According to [docs](https://www.erlang.org/doc/man/ets.html#lookup-2), the first parameter is a table, and the second is the key. If it isn't a typo I recommend renaming the key variable.

Reply:
I don't know I copied it from `event_channele.ex` but I will look at it.